datafusion/common-runtime/Cargo.toml

```diff
@@ -37,3 +37,6 @@ path = "src/lib.rs"
 
 [dependencies]
 tokio = { workspace = true }
+
+[dev-dependencies]
+tokio = { version = "1.36", features = ["rt", "rt-multi-thread", "time"] }
```

Comment:
How about we just add this `rt-multi-thread` feature to the workspace dependency here: https://github.com/apache/datafusion/blob/e6e1eb229440591263c82bb2b913a4d5a16f9b70/Cargo.toml#L141

(it already gets used a bunch so having different feature flags just means the crates end up recompiling a bunch unecessairly during development)

Reply:
Turns out the the `rt-multi-thread` feature is not supported on wasm, and [CI was failing](https://github.com/influxdata/arrow-datafusion/actions/runs/10497744745/job/29081242552).

```
error: Only features sync,macros,io-util,rt,time are supported on wasm.
   --> /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/tokio-1.39.3/src/lib.rs:467:1
    |
467 | compile_error!("Only features sync,macros,io-util,rt,time are supported on wasm.");
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```